Styling profile marcos

alvarezGarciaMarcos opened the pull request:
I would like to know your opinion on the design, I am sure it can be improved in a lot of ways.

alvarezGarciaMarcos committed:
Merge pull request #49 from Arquisoft/dev

Routing, Profile...

alvarezGarciaMarcos committed:
Changed homepage parameter in package.json

alvarezGarciaMarcos committed:
Styling of the profile

themrcesi commented on src/components/user/myProfile/FriendList.js:
```diff
@@ -15,14 +16,14 @@ export default function MyProfile() {
 		{
 			(item, i) =>
-				<Card key={uniqueKey.toString()} onClick={() => openProfile(`${item}` )}>
+				<Card id={uniqueKey.toString()} onClick={() => openProfile(`${item}` )}>
```
Here it needs to be key, not id. Key propierty is the one that must be unique for each element of a map.

themrcesi reviewed (changes requested):
I´d also increase a little bit the padding for the general MyProfile component. But this is just my personal opinion.

themrcesi commented:
And now it looks so much better!

themrcesi commented:
if you open the console in myProfile, there are some warnings related to the navbar i think, about an <a> tag.

themrcesi commented:
and also the activeclassname propierty, we need to think what to do with it.

alvarezGarciaMarcos committed:
Added modifications

I added the modifications purposed in the issue: the padding for the profile, the removal of the activeclass in the navbar, and also the modification
of the word 'id' for 'key' in the FriendList.js . The tag that @themrcesi talked about is from a component that I used, so I cannot fix that.
